test/unit/win_interface_library/win_interface_library_analysis_test.bzl

```diff
@@ -36,6 +36,12 @@ def win_interface_library_analysis_test_suite(name):
         target_compatible_with = ["@platforms//os:windows"],
     )
 
+    native.filegroup(
```

Comment:
Can you add a filegroup for a `rust_binary` target as well?

Reply:
sure can do! now let's see if the tests still pass 🤞🏼

Reply:
ah derp I added it to the wrong binary type, give me a sec.